makefile

```diff
@@ -662,7 +662,8 @@ status info:
 	$(info MFEM_MPI_NP            = $(MFEM_MPI_NP))
 	@true
 
-ASTYLE = astyle --options=$(SRC)config/mfem.astylerc
+ASTYLE_BIN = astyle
+ASTYLE = $(ASTYLE_BIN) --options=$(SRC)config/mfem.astylerc
 FORMAT_FILES = $(foreach dir,$(DIRS) $(EM_DIRS) config,"$(dir)/*.?pp")
```

Comment:
This is OK in general, but why will the `astyle` binary have a different name?

Reply:
When I was using astyle in Gitlab, I wanted to specify the full path to the desired version on astyle.
This is now useless and has been reverted.